### PR TITLE
Change password

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**

### CODE REVIEW

File: VulnerableApplication/Global.asax.cs [hunk 1]
```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:17]: Add or update the header of this file.

File: VulnerableApplication/Global.asax.cs [hunk 2]
```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
...
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public string ChangePassword { get; set; }
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL]3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL]7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL]Add or update the header of this file.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL] 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL] 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: [CODE_SMELL] Add or update the header of this file.
